src/languages/vala.js

```diff
@@ -21,8 +18,11 @@ function(hljs) {
         'public private protected internal ' +
         // Other
         'using new this get set const stdout stdin stderr var',
+      type:
+        'char uchar unichar int uint long ulong short ushort int8 int16 int32 int64 uint8 ' +
+        'uint16 uint32 uint64 float double bool struct enum string void',
```

Comment:
Alright. I'll close this PR then considering there isn't much else besides this. Thanks for your time :)